N Lab Name/QuickSort.py

```diff
@@ -0,0 +1,17 @@
+numbersArr = [2,5,1,6,4,15,3]
```

Comment:
0. Rename the folder to 1 Quick Sort instead of N Lab Name. N Lab Name - is just a naming template, do not use it as is. 

1. The main idea of this algorithm implementation is recursion (when you call a function right from itself). 
Almost always it's a bad practice and developers try to avoid it. It's easy to break such code. Take a look on this article for the explanation:
http://qaru.site/questions/34454/what-is-the-maximum-recursion-depth-in-python-and-how-to-increase-it

2. We'll left this variant with a recursion only for educational purposes. Read about Hungarian Notation and refactor your code to avoid it. It's old-fashioned, no need to specify type directly in the variable name. https://ru.m.wikipedia.org/wiki/%D0%92%D0%B5%D0%BD%D0%B3%D0%B5%D1%80%D1%81%D0%BA%D0%B0%D1%8F_%D0%BD%D0%BE%D1%82%D0%B0%D1%86%D0%B8%D1%8F

3. To make this program useful we should allow users to sort their custom arrays, which are located in file. Create file with: 1 5 12 46 75 4 3 8 90 (example, you will need a lot more) . After running console you must see - Sorted Array is: 1 3 4 5 8 12 46 75 90. Doing that part, create separate function "readArrayFromFile" which gets file path and return array. 
Create 100 numbers using this service https://www.random.org/integers/?num=100&min=1&max=100&col=100&base=10&format=html&rnd=new

5. Create 5000 now and make sure your program fails.